ports/osgearth/vcpkg.json

```
@@ -1,15 +1,12 @@
 {
   "name": "osgearth",
-  "version": "3.7",
-  "port-version": 1,
-  "description": "osgEarth - Dynamic map generation toolkit for OpenSceneGraph Copyright 2021 Pelican Mapping.",
+  "version": "3.7.2",
+  "description": "osgEarth - 3D Maps for OpenSceneGraph / C++14",
   "homepage": "https://github.com/gwaldron/osgearth",
   "license": "LGPL-3.0-or-later",
   "supports": "!(arm | x86 | wasm32 | xbox)",
```

I wonder if arm64 isn't supported. It could be enabled with
```suggestion
  "supports": "!(arm32 | x86 | wasm32 | xbox)",
```

---

Sure, we'll try it.